content/user/ui/report/Vulnerability-trends.md

```diff
@@ -0,0 +1,71 @@
+<!--
+title: "Vulnerability Trends"
+description: "Overview of vulnerability trend reports"
+tags: "TeamServer application vulnerability PDF report"
+-->
+ 
+Vulnerability management is a vital responsibility of any security team. Recognizing the vulnerabilities your applications face and how well they're being managed can help you understand your security posture.
+
+The report presents data about newly discovered vulnerabilities as well as accumulated total vulnerabilities in comparison to remediations. Focus on the details that you care about by using robust filtering options to look at data over any applications, servers, Assess rules or by vulnerability severity. Contrast can also present data in various timeframes, ranging from the last week to the last year. The customized results allow you to prioritize efforts toward vulnerability remediation.
```

Comment:
Maybe start this sentence as "The **vulnerability trend** report presents data..." since it's the first time we are referencing it. I know the title tells you what report we are referring to, but as I read through this, just saying "The report" sounded a little too vague to me. Or, if you have any other ideas, that's cool too!